### DISCUSSION TITLE
Downloads

### DISCUSSION THREAD

**Discussion by SupreemeGuy** [arichornlover/uYouEnhanced#244] [Q&A] [answered]
Hello, is it possible to access the downloaded videos? I can't find the uYou folder through the default iOS files app, is there any other way?

**Comment by ego-lay-atman-bay** [comment, marked as answer]
You can only find it if you have filza (and if you have filza installed, there's a button in the uYou options to go to the folder in filza). Now, I will have to warn you, you can only get filza by jailbreaking, or through trollstore, which only works on ios 15. Now, there is indeed another way. You can change the download location to the camera roll in the uYou settings, which saves the videos to your photos.

There is actually also another way. In the uYou tab, tap on the 3 dotes on a downloaded video, and tap share. This will open a share prompt, which you can tap save video. I do also want to mention that, at least on ipad, this share prompt is a bit broken. The size is all wonky, so there's barely any space to see the options, but there is enough space to scroll down and select save video.
